backend/activity_log/tests/test_api_views.py

```diff
@@ -58,3 +58,40 @@ def test_list_api_filter_by_event_type(self):
 
         self.assertEqual(response.data[0]["total"], 1)
         self.assertEqual(response.data[0]["date"], str(self.today.date()))
+
+
+class TestActivityLogEventTypeViewSet(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "activitylogeventtype-list"
+    SUPPRESS_TEST_GET_LIST = True
+    SUPPRESS_TEST_POST_TO_CREATE = True
+
+    def setUp(self):
+        self.today = timezone.now()
+
+        self.entry1 = factories.EventTypeFactory()
+        self.entry1 = factories.LogEntryFactory()
+        self.entry1.timestamp = self.today
+        self.entry1.save()
+
+        self.entry2 = factories.EventTypeFactory()
+        self.entry2 = factories.LogEntryFactory()
+        self.entry2.timestamp = self.today
+        self.entry2.save()
+
+    def test_list_api_filter_by_event_type(self):
+        url = f"{self.get_list_url()}?event_type={self.entry1.event_type}"
+        response = self.client.get(url)
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(self.entry1.timestamp, self.today)
+
+    def test_list_api_filter_by_effected_user(self):
+        url = f"{self.get_list_url()}?effected_user={self.entry1.effected_user.id}"
+        response = self.client.get(url)
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(self.entry1.timestamp, self.today)
+
+    def test_list_api_filter_by_actor_user(self):
+        url = f"{self.get_list_url()}?actor_user={self.entry1.actor_user.id}"
+        response = self.client.get(url)
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(self.entry1.timestamp, self.today)
```

Comment:
in your setUp you create two entries but in your tests you only use one of them - is there a reason for this?